argo-cd-apps/base/member/infra-deployments/nvme-storage-configurator/nvme-storage-configurator.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: nvme-storage-configurator
+spec:
+  generators:
+    - merge:
+        mergeKeys:
+          - nameNormalized
+        generators:
+          - clusters:
+              values:
+                sourceRoot: configs/nvme-storage-configurator
```

Comment:
I think that we should follow the pattern used in the other application sets so we can have different versions of this app in prod and stage (for testing purposes).

Reply:
changes made for handling staging and production overlays